.circleci/config.yml

```diff
@@ -896,19 +896,69 @@ commands:
       - slack/notify:
           branch_pattern: main
           event: fail
-          mentions: '@$CIRCLE_USERNAME'
           custom: |
             {
+              "text": "CircleCI job failed.",
               "blocks": [
-                {
-                  "type": "section",
-                  "fields": [
-                    {
+                  {
+                      "type": "header",
+                      "text": {
+                          "type": "plain_text",
+                          "text": "Job Failed. :red_circle:",
+                          "emoji": true
+                      }
+                  },
+                  {
+                      "type": "section",
+                      "fields": [
+                          {
+                              "type": "mrkdwn",
+                              "text": "*Job*: ${CIRCLE_JOB}"
+                          }
+                      ]
+                  },
+                  {
+                      "type": "section",
+                      "fields": [
+                          {
+                              "type": "mrkdwn",
+                              "text": "*Project*: $CIRCLE_PROJECT_REPONAME"
+                          },
+                          {
+                              "type": "mrkdwn",
+                              "text": "*Branch*: $CIRCLE_BRANCH"
+                          },
+                          {
+                              "type": "mrkdwn",
+                              "text": "*Author*: @${CIRCLE_USERNAME}"
+                          }
+                      ],
+                      "accessory": {
+                          "type": "image",
+                          "image_url": "https://assets.brandfolder.com/otz5mn-bw4j2w-6jzqo8/original/circle-logo-badge-black.png",
```

Comment:
Where this asset being served from? It is a storage platform that we use.

Reply:
It's from default template of Slack's orb